cmd/ib-sriov-cni/main.go

```diff
@@ -388,41 +453,18 @@ func cmdDel(args *skel.CmdArgs) (retErr error) {
 	}
 	defer netns.Close()
 
+	if !netConf.IsVFDevice {
```

Comment:
netconf is loaded from cache. currently this field is not persisted.

Reply:
please also think of a case where CNI Add was called with a binary that did not contain this change, then the CNI plugin was updated and CMD Del command was called with this code.

in that case even if we persisted the value it would default to false in this case.

perhaps its worth to check if the device is PF or VF during runtime here as well